src/data/sql-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, which of the following is the correct syntax for creating a table?",  
+    Answer: "CREATE TABLE table_name (column1_name datatype, column2_name datatype)",
+    Distractor1: "CREATE TABLE table_name (column1_name, column2_name )",
+    Distractor2: "UPDATE TABLE table_name (column1_name datatype, column2_name datatype)",
+    Distractor3: "CREATE NEW TABLE table_name (column1_name datatype, column2_name datatype)",
```

Comment:
We want to try to keep the answers on the shorter side when possible

```suggestion
    Answer: "CREATE TABLE table_name (column1_name datatype, ...)",
    Distractor1: "CREATE TABLE table_name (int main void column1_name)",
    Distractor2: "UPDATE TABLE table_name (column1_name datatype, ...)",
    Distractor3: "CREATE NEW TABLE table_name (column1_name datatype,...)",
```